packages/bruno-app/src/components/Sidebar/Collections/Collection/CollectionItem/GenerateCodeItem/CodeView/index.js

```diff
@@ -20,13 +24,20 @@ const CodeView = ({ language, item }) => {
   }
 
   return (
-    <CodeEditor
-      readOnly
-      value={snippet}
-      font={get(preferences, 'font.codeFont', 'default')}
-      theme={storedTheme}
-      mode={lang}
-    />
+    <>
+      <div style={{ display: 'flex', justifyContent: 'flex-end' }}>
+        <CopyToClipboard text={snippet} onCopy={() => toast.success('Copied to clipboard!')}>
+          <FontAwesomeIcon icon={faCopy} style={{ cursor: 'pointer' }} />
```

Comment:
Based on `contributing.md` file Bruno is using Tabler icons, so probably should be replaced with Tabler icon to not mix icons from different collections.

Reply:
agree